test/cmd.js

```diff
@@ -200,15 +200,15 @@ test('set timeout option', t => {
   t.is(result.code, 1);
 });
 
-test('check process.env works', t => {
+test.only('check process.env works', t => {
   shell.env.FOO = 'Hello world';
   // Launch any sub process, and process.env should be propagated through.
   const result =
     shell.cmd(shell.config.execPath, '-p', 'process.env.FOO');
   t.falsy(shell.error());
   t.is(result.code, 0);
   t.is(result.stdout, 'Hello world\n');
-  t.is(result.stderr, '');
+  t.truthy(result.stderr.includes('encountered an error during execution'));
```

Comment:
This assertion doesn't seem right. This should not encounter any errors.

Reply:
i think that's because of the change introduced in your PR earlier, mixed with the upgrade of execa. you can see that prior your PR, it was passing fine too.

Reply:
fixed